### ISSUE TITLE
Installation error on Windows 8.1!

### ISSUE THREAD

**Issue by wealstarr** [intel/openvino-plugins-ai-audacity#352]
Hello,

When I try to install the latest version I get this error:
![Capture](https://github.com/user-attachments/assets/5ee0dbaf-4cfe-4f57-a33b-cbdd4181139e)

Host: Windows 8.1 x64

Could you please help me to install the plugins, what am I doing wrong?

Any help will be greatly appreciated.

**Comment by dd333** [comment]
Enough free space one your OS partition?

**Comment by alglove** [comment]
I know this is an old comment, but "Error sending data: (12175) A security error occurred" on old Windows computers is often due to outdated security protocols being used for WinHTTP requests. In other words, when you are trying to download the installation files, you cannot connect to the server because of an outdated setting on your computer.

On your computer, go to Control Panel --> Internet Options --> Advanced.  Scroll down to the "Security" section of the list and make the following changes:

Use SSL (anything before 2.0):  uncheck
Use SSL 2.0:  uncheck
Use SSL 3.0:  uncheck
Use TLS 1.0:  uncheck
Use TLS 1.1:  uncheck
Use TLS 1.2:  check
Use TLS 1.3:  check

Basically, most websites these days use TLS 1.2 or newer when making secure connections.  SSL is generally avoided.  Since you have an older computer, you many not have the proper settings to connect to the download server.

You may have to reboot the computer to for the settings to take effect.  I am not sure.

Those are the settings from my Windows 11 computer.  I am not sure how much overlap there is between Windows 8.1 and Windows 11.  I think Windows 8.1 has TLS 1.2, but if it doesn't, try TLS 1.1.
